content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md

```diff
@@ -32,7 +32,7 @@ test: no
     {{< /text >}}
 
     {{< tip >}}
-    如果您是一位讲师，可以为每个参与者分配独立的命名空间。本教程支持多个参与者在不同的命名空间下同时运行。
+    如果您是管理员，您可以为每个用户分配一个独立的命名空间。本教程支持多个用户同时在多个命名空间中进行工作。
```

Comment:
You may need to change the en first if you believe the prospective of `admin` is better.